cmd/subcommands/transfer.go

```diff
@@ -131,12 +132,28 @@ func handlerForTransaction(txLog *transactionLog) error {
 		return err
 	}
 
+	var gLimit uint64
+	if gasLimit == "" {
+		gLimit, err = core.IntrinsicGas([]byte(""), false, true, false)
+		if err != nil {
```

Comment:
handle the error with `handlerForError` so that batched txs can get errors aswell.